device-scripts/speedtest.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
```

Comment:
Do we need/want to duplicate the speed test logic here?

I would prefer having it only in one since place to avoid maintenance issues.